src/Page.js

```diff
@@ -332,22 +333,23 @@ Page.prototype.collectHeadingsAndKeywords = function () {
   this.headings = {};
   this.keywords = {};
   // Collect headings and keywords
-  this.collectHeadingsAndKeywordsInContent($(`#${CONTENT_WRAPPER_ID}`).html(), null, false);
+  this.collectHeadingsAndKeywordsInContent($(`#${CONTENT_WRAPPER_ID}`).html(), null, false, []);
 };
 
 /**
  * Records headings and keywords inside content into this.headings and this.keywords respectively
  * @param content that contains the headings and keywords
  */
-Page.prototype.collectHeadingsAndKeywordsInContent = function (content, lastHeading, excludeHeadings) {
+Page.prototype.collectHeadingsAndKeywordsInContent = function (content, lastHeading, excludeHeadings,
+                                                               callStack) {
```

Comment:
`callStack` sounds a bit weird (because it is not directly relevant to the pages), name it `sourceTraversalStack`?

Reply:
Just tested on CS2103T website:
```
info: Building assets...
info: Assets built
info: Generating pages...
[=========================================================================================================================================================] 714 / 714 pages buil
info: Pages built
info: Website generation complete! Total build time: 170.116s
http://0.0.0.0:8080 is already in use. Trying another port.
info: Site data built
info: Serving "/workspace/cs2103-website/_site" at http://127.0.0.1:43455
```
All's good :D

Reply:
callStack -> sourceTraversalStack: Changed. Thanks!